tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/RepositoryScriptService.cs

```diff
@@ -0,0 +1,164 @@
+using System.Collections.Specialized;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Azure.Sdk.Tools.Cli.Helpers;
+
+namespace Azure.Sdk.Tools.Cli.Services;
+
+public interface IRepositoryScriptService
+{
+    Task<string> GetCommand(string commandName, string packagePath, CancellationToken ct);
+    Task<bool> HasImplementation(string commandName, string packagePath, CancellationToken ct);
+    Task<(bool invoked, ProcessResult result)> TryInvoke(
+        string commandName,
```

Comment:
CommandName is...path to a powershell script? An exe?

Reply:
It's a script path. I'll update as this is unclear.